src/main/java/org/gridsuite/modification/server/dto/MassiveEquipmentsModificationsInfos.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.dto;
+
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import com.powsybl.commons.reporter.Reporter;
+import com.powsybl.commons.reporter.ReporterModel;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.SuperBuilder;
+import org.gridsuite.modification.server.ModificationType;
+import org.gridsuite.modification.server.dto.annotation.ModificationErrorTypeName;
+import org.gridsuite.modification.server.entities.ModificationEntity;
+import org.gridsuite.modification.server.entities.equipment.modification.MassiveEquipmentsModificationsEntity;
+import org.gridsuite.modification.server.modifications.AbstractModification;
+import org.gridsuite.modification.server.modifications.MassiveEquipmentsModifications;
+
+import java.util.List;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+@SuperBuilder
+@NoArgsConstructor
+@Getter
+@Setter
+@Schema(description = "Massive equipments modifications infos")
+@JsonTypeName("MASSIVE_EQUIPMENTS_MODIFICATIONS")
+@ModificationErrorTypeName("MASSIVE_EQUIPMENTS_MODIFICATIONS_ERROR")
+public class MassiveEquipmentsModificationsInfos extends ModificationInfos {
+    @Schema(description = "equipments modifications")
+    private List<ModificationInfos> modifications;
```

Comment:
I don't remember, is this enough for our use cases or do we want more structure ? For exemple, grouping multiple modifications in the same "row" of the tabular view ?

```
GeneratorModification(G1, SETX=X1, SETY=Y1)
GeneratorModification(G2, SETX=X2, SETY=Y2)

G1    SETX1  SETY1
G2    SETX2  SETY2
```

But many we want to correlate some other modification with these and we have no hardcoded way of knowing which goes to which ?

```
GeneratorModification(G1, SETX=X1, SETY=Y1)
GeneratorModification(G2, SETX=X2, SETY=Y2)
VoltageLevelModification(VLG1, SETZ=Z1)
VoltageLevelModification(VLG2, SETZ=Z2)

# Do we want this
G1    SETX1  SETY1  SETZ1
G2    SETX2  SETY2  SETZ2

# or is this enough ?
G1      SETX1  SETY1
G2      SETX2  SETY2
VLG1    SETZ1
VLG2    SETZ2
```
Note, if the correlation is always recoverable from the data then we don't need additional structure. For exemple if the data shares an ID, the client can reconstruct the tabular view themselves

Reply:
I don't think there is a need for grouping multiple modifications in the same row : in that case, the modifications will be shown in different tables